client/src/main/java/io/pravega/client/netty/impl/ClientConnectionInboundHandler.java

```diff
@@ -66,7 +66,7 @@ public void channelRegistered(ChannelHandlerContext ctx) throws Exception {
     }
 
     @Override
-    public void channelUnregistered(ChannelHandlerContext ctx) throws Exception {
+    public void channelInactive(ChannelHandlerContext ctx) throws Exception {
```

Comment:
I don't think there is documentation explaining this well, could we please add a comment block to explain how these channel methods are called and what we expect to happen?